test/test.tape.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+const InjectRevisions = require('../index.js');
+const test = require('tape');
+const async = require('async');
+const fs = require('fs');
+
+test('can load the class', (t) => {
+  t.plan(1);
+  const task = new InjectRevisions();
+  t.deepEqual(task.options, {
+    mappingPath: 'assets.json',
+    startTag: '<!-- clientkit:(.*?) -->',
+    endTag: '<!-- clientkit:end -->',
+    uiPath: '/ui/',
+    files: ['index.html']
+  });
+});
+
+const file = 'test/fixtures/index1.html';
+
+test('can replace the references in a file reference', (t) => {
+  t.plan(3);
+  async.autoInject({
+    task: (done) => {
+      const task = new InjectRevisions('revisions', {
+        files: [file]
+      });
+      task.execute(done);
+    },
+    verifyFirstOutput: (task, done) => {
+      const content = fs.readFileSync(file).toString();
+      t.equal(content.indexOf('<link rel="stylesheet" href="/ui/common-abcdefg.css">', `<head>
+    <!-- clientkit:common.css -->
+    <link rel="stylesheet" href="/ui/common-abcdefg.css">
+    <!-- clientkit:end -->
+    <!-- clientkit:script.js -->
+    <script type="application/javascript" src="/ui/script-abcdefg.js">
+    <!-- clientkit:end -->
+    </head>`) > -1, true);
+      done();
+    },
+    task2: (verifyFirstOutput, done) => {
+      const task = new InjectRevisions('revisions', {
+        files: [file]
+      });
+      task.execute(done);
+    },
+    verifySecondOutput: (task2, done) => {
+      const content = fs.readFileSync(file).toString();
+      t.equal(content.indexOf('<link rel="stylesheet" href="/ui/common-abcdefg.css">', `<head>
```

Comment:
can we just put an output.html in fixtures with the expected output and compare the two?